jobs/delorean/jenkinsfiles/branch/ga/Jenkinsfile

```diff
@@ -73,6 +80,25 @@ node {
         }
     }
 
+    stage('Create Jira Upgrade Issue') {
+        when(sourceChanges) {
+          gitCheckoutRepo(installationGitUrl, targetBranch, githubSSHCredentialsID, 'installation')
```

Comment:
Is this just checking out the target version of the installation repo? So it just switches branches?

Reply:
Yes, it checks out the `master` branch so we could get the product version that's in master.